ParcelDistributionCenter/ParcelDistributionCenter.Logic/Services/PackageServices.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ParcelDistributionCenter.Model.Enums;
+using ParcelDistributionCenter.Model.Models;
+using ParcelDistributionCenter.Web.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace ParcelDistributionCenter.Logic.Services
+{
+    public class PackageServices : IPackageServices
+    {
+        private readonly IMemoryRepository _memoryRepository;
+
+        public PackageServices(IMemoryRepository memoryRepository)
+        {
+            _memoryRepository = memoryRepository;
+        }
+
+        public List<Package> GetCourierPackages(string courierId)
+        {
+            var packages = _memoryRepository.PackagesList.Where(p => p.CourierId == courierId).ToList();
+
+            return packages;
+        }
+
+        public List<AssignPackagesVM> GetUnassignedPackages()
+        {
+            Courier unknownCourier = new Courier("Unassigned", "Unknown", "Unknown", "Unknown", "Unknown");
```

Comment:
To na pewno bym osobiście jakoś przerobił, żeby chcociaż ten "Unknown" był w jakiejś zmiennej, bo łatwo o literówke.

Reply:
Jeszcze też do końca nie rozumiem, czemu musi być jakiś unknown courier.